packages/yoga/src/Card/native/PlanCard/List.jsx

```diff
@@ -72,11 +87,13 @@ ListItem.propTypes = {
   /** if provided displays a button below the item text. It accepts all button
    * element props */
   buttonProps: shape({}),
+  variant: string,
```

Comment:
I believe it would be nice to document this property (by commenting a line before like other examples) as also adding a prop validation to check if it is included in our color tokens, what do you think?

Reply:
Nice, I set a little commentary above the props, about the validation, in the line 51 it will try to get the color from tokens, if it not found any color, the variable `itemColor` will be `undefined`, so in styled component `ItemText` will set a default color.

Reply:
Makes sense, but what do you think about throwing a warning in this case so the developer knows that they're giving an invalid value?

E.g.:
```jsx
<Comp variant="deprecated-token" />
```

Since `variant` isn't undefined but the token doesn't exist, a warning such as 'Invalid token "deprecated-token", defaulting to "default-value-name"' could help one debugging what they did wrong and why their token is being overridden.